strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/ConfigurationManagementServiceImpl.java

```diff
@@ -187,28 +187,30 @@ public Repository getRepository(String storageId,
     }
 
     @Override
-    public List<Repository> getGroupRepositoriesContaining(String repositoryId)
+    public List<Repository> getGroupRepositoriesContaining(String storageId,
```

Comment:
Yeah, I think you're right... I initially did it like this because I was thinking about the case where multiple storages could have repositories with the same ID. 
I have the feeling that we'll probably still need the old method at some point as well, but let's leave it as it is.